service/src/main/java/org/ehrbase/service/EhrServiceImp.java

```diff
@@ -84,6 +87,8 @@
 @Transactional()
 public class EhrServiceImp extends BaseServiceImp implements EhrService {
     public static final String DESCRIPTION = "description";
+    public static final String EHR_NOT_FOUND_WITH_SUPPLIED_SUBJECT =
```

Comment:
We should maybe change the message to say that the `partyId` is used by a different EHR in the same `partyNamespace`